dali/benchmark/one_hot_bench.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021, Maksymilian Grochowski. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+#include "dali/benchmark/operator_bench.h"
+#include "dali/benchmark/dali_bench.h"
+
+namespace dali {
+
+static void OneHotGPUArgs(benchmark::internal::Benchmark *b) {
+  for (int batch_size = 256; batch_size >= 1; batch_size /= 2) {
+    int h = 64, w = h;
+    int axis = -1;
+    int numClasses = 500;
```

Comment:
Nitpick: we don't use camelCase.
```suggestion
    int num_classes = 500;
```

BTW - while I consider testing a configuration with a large number of classes and few entries to be essential, I have second thoughts about _additionally_ running a test with a large input and few classes. This could (arguably) be useful for encoding labels for segmentation.

I'm approving this PR anyway, you can address this comment at your leisure.